doc/how_to/make_a_sorting.rst

```diff
@@ -0,0 +1,89 @@
+Make your own Sorting
+=====================
+
+Why make a :code:`Sorting`?
+
+The :code:`Sorting` object is one of the core objects within the SpikeInterface library
+along with :code:`Recording` and :code:`SortingAnalyzer`. Although SpikeInterface has
+many tools for reading sorting formats you may have some data in a nonstandard format
+(e.g. old csv file). If this is the case you would need to make your own :code:`Sorting`.
+
+At a fundamental level the :code:`Sorting` is a series of spike times and a series of
+labels for each spike along with some associated metadata. Thus by providing this
+information you can easily make a :code:`Sorting` object to be used for various analyses
+(e.g. correlograms or for generating a :code:`SortingAnalyzer`)
+
+
+Making a :code:`Sorting`
+------------------------
+
+For most formats the :code:`Sorting` is automatically generated. For example one could do
+
+.. code-block:: python
+
+    from spikeinterface.extractors import read_kilosort, read_phy
+
+    # For kilosort/phy files we can use either reader
+    ks_sorting = read_kilosort('path/to/folder')
+    phy_sorting = read_phy('path/to/folder')
+
+This :code:`Sorting` contains important information about your spike trains including
+the spike times (i.e. when the neurons were actually firing) the unit labels (i.e.
+who the spikes belong to. Also called cluster ids by some sorters), the unit ids (the unique
+set of unit labels) and the sampling_frequency. To make your own :code:`Sorting` object you can
+use :code:`NumpySorting`. It is important to note if you are new to spike trains that they
```

Comment:
```suggestion
use :code:`NumpySorting`. It is important to note that in SpikeiInterface spike trains are handled internally in samples/frames rather than in seconds and we use the sampling frequency to ...
```

I think something along these lines is more precise. What do you think?